onnxruntime/python/tools/transformers/models/llama/requirements-cpu.txt

```diff
@@ -1,2 +1,3 @@
 -r requirements.txt
-onnxruntime>=1.16.2
\ No newline at end of file
+onnxruntime>=1.16.2
+psutil
```

Comment:
If it is needed for both cpu and gpu, we can add psutil to requirements.txt

Reply:
done.